charts/argo-cd/templates/argocd-server/ingress.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if and .Values.server.ingress.enabled (eq .Values.server.ingress.controller "generic") }}
 {{- $insecure := index .Values.configs.params "server.insecure" | toString -}}
-{{- $servicePort := eq $insecure "true" | ternary .Values.server.service.servicePortHttp .Values.server.service.servicePortHttps -}}
+{{- $servicePort := eq $insecure "true" | ternary .Values.server.service.servicePortHttps .Values.server.service.servicePortHttp -}}
```

Comment:
Are you sure this is incorrect? Based on the docs if true first parameter is used and we're checking for insecure :thinking: 
https://helm.sh/docs/chart_template_guide/function_list/#ternary

Reply:
Yes, sorry mistake in my configuration.